From 30ef8621fe7f78259a30bf406d4a30185239eeac Mon Sep 17 00:00:00 2001
From: YOUNG HO CHA <ganadist@gmail.com>
Date: Wed, 31 Aug 2016 09:49:15 +0900
Subject: [PATCH] sepolicy: allow execmod for cameraserver

Signed-off-by: YOUNG HO CHA <ganadist@gmail.com>
---
 domain.te | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/domain.te b/domain.te
index 1ac33f1..248676e 100644
--- a/domain.te
+++ b/domain.te
@@ -432,6 +432,7 @@ neverallow * {
   -apk_data_file
   -app_data_file
   -asec_public_file
+  -system_file
 }:file execmod;
 
 # Do not allow making the stack or heap executable.
@@ -441,7 +442,7 @@ neverallow * self:process { execstack execheap };
 
 # prohibit non-zygote spawned processes from using shared libraries
 # with text relocations. b/20013628 .
-neverallow { domain -appdomain } file_type:file execmod;
+neverallow { domain -appdomain -camera -cameraserver } file_type:file execmod;
 
 neverallow { domain -init } proc:{ file dir } mounton;
 
-- 
2.9.3

