From dbaf04bca2978d52d387c2bc04e64bff3fd6f359 Mon Sep 17 00:00:00 2001
From: YOUNG HO CHA <ganadist@gmail.com>
Date: Wed, 31 Aug 2016 09:58:16 +0900
Subject: [PATCH 1/2] libc_pthread: do not use clang on mako build

To allow to run sensor service

Signed-off-by: YOUNG HO CHA <ganadist@gmail.com>
---
 libc/Android.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/libc/Android.mk b/libc/Android.mk
index 1ca84c1..34b14ee 100644
--- a/libc/Android.mk
+++ b/libc/Android.mk
@@ -1071,7 +1071,12 @@ LOCAL_CONLYFLAGS := $(libc_common_conlyflags)
 LOCAL_CPPFLAGS := $(libc_common_cppflags) -Wold-style-cast
 LOCAL_C_INCLUDES := $(libc_common_c_includes)
 LOCAL_MODULE := libc_pthread
+ifeq ($(PRODUCT_DEVICE),mako)
+LOCAL_CLANG := false
+else
 LOCAL_CLANG := $(use_clang)
+endif
+
 LOCAL_ADDITIONAL_DEPENDENCIES := $(libc_common_additional_dependencies)
 LOCAL_CXX_STL := none
 LOCAL_SYSTEM_SHARED_LIBRARIES :=
-- 
2.9.3

