From 5f8dd0863d19aed8d15309c7907bc3b0b718d1ff Mon Sep 17 00:00:00 2001
From: YOUNG HO CHA <ganadist@gmail.com>
Date: Wed, 31 Aug 2016 11:15:26 +0900
Subject: [PATCH 2/2] linker: allow text relocation to mm-qcamera-daemon for
 mako

Signed-off-by: YOUNG HO CHA <ganadist@gmail.com>
---
 linker/linker.cpp | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/linker/linker.cpp b/linker/linker.cpp
index 39aa2ca..5b044a7 100644
--- a/linker/linker.cpp
+++ b/linker/linker.cpp
@@ -49,6 +49,7 @@
 #include "private/KernelArgumentBlock.h"
 #include "private/ScopedPthreadMutexLocker.h"
 #include "private/ScopeGuard.h"
+#include "private/android_filesystem_config.h"
 
 #include "linker.h"
 #include "linker_block_allocator.h"
@@ -3903,6 +3904,10 @@ bool soinfo::prelink_image() {
   return true;
 }
 
+static bool __is_user(uid_t uid) {
+  return uid == getuid();
+}
+
 bool soinfo::link_image(const soinfo_list_t& global_group, const soinfo_list_t& local_group,
                         const android_dlextinfo* extinfo) {
 
@@ -3924,7 +3929,7 @@ bool soinfo::link_image(const soinfo_list_t& global_group, const soinfo_list_t&
 #if !defined(__LP64__)
   if (has_text_relocations) {
     // Fail if app is targeting sdk version > 22
-    if (get_application_target_sdk_version() > 22) {
+    if (get_application_target_sdk_version() > 22 && !__is_user(AID_CAMERA)) {
       PRINT("%s: has text relocations", get_realpath());
       DL_ERR("%s: has text relocations", get_realpath());
       return false;
-- 
2.9.3

