From 9a09881e7c19a54cca5fabbe0b4165c15d3d0594 Mon Sep 17 00:00:00 2001
From: YOUNG HO CHA <ganadist@gmail.com>
Date: Tue, 30 Aug 2016 09:20:34 +0900
Subject: [PATCH] init.rc: load product specific rc file

Signed-off-by: YOUNG HO CHA <ganadist@gmail.com>
---
 rootdir/init.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 3466dce..0caea3e 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -7,6 +7,7 @@
 import /init.environ.rc
 import /init.usb.rc
 import /init.${ro.hardware}.rc
+import /init.${ro.product.name}.rc
 import /init.usb.configfs.rc
 import /init.${ro.zygote}.rc
 
-- 
2.9.3

