From 7452ca1bdc19558467fe50d790392bc175fe0ca6 Mon Sep 17 00:00:00 2001
From: YOUNG HO CHA <ganadist@gmail.com>
Date: Mon, 5 Sep 2016 17:07:07 +0900
Subject: [PATCH] surfaceflinger: add service as foreground task

Signed-off-by: YOUNG HO CHA <ganadist@gmail.com>
---
 services/surfaceflinger/surfaceflinger.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/services/surfaceflinger/surfaceflinger.rc b/services/surfaceflinger/surfaceflinger.rc
index 2b4ea2a..f75a7df 100644
--- a/services/surfaceflinger/surfaceflinger.rc
+++ b/services/surfaceflinger/surfaceflinger.rc
@@ -4,3 +4,4 @@ service surfaceflinger /system/bin/surfaceflinger
     group graphics drmrpc readproc
     onrestart restart zygote
     writepid /sys/fs/cgroup/stune/foreground/tasks
+    writepid /dev/cpuset/foreground/tasks
-- 
2.9.3

